utilities/virt.py

```
@@ -1550,27 +1550,33 @@ def get_guest_os_info(vmi):
 
 
 def get_windows_os_dict(windows_version):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)